bin/validate.js

```diff
@@ -59,7 +59,9 @@ async function requireDimensions(path) {
         reject(new Error(`load image ${path}: ${err}`));
```

Comment:
Please revert changes in this file

Reply:
I've reverted the changes in the file but it still shows the last line as modified, are you able to change it?